<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClassSaveDlg</class>
 <widget class="QDialog" name="CClassSaveDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CClassSaveDlg
{
background-color:#f1f3f6;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget
{
background-color:#ffffff ;
border-radius:50; 
}</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton
{
color:#a7b0ca;
  background-color: rgba(0, 0, 0, 0);
  border: none; 
}

#pushButton:hover{
color:#dd3a5f;
}</string>
    </property>
    <property name="text">
     <string>&lt; 返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label
{
color:#1a263c;
}</string>
    </property>
    <property name="text">
     <string>课程信息输入</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit
{
/*border:none;*/
border: 1px solid #e5e7e9;
border-left:none;
border-top-right-radius:25;
border-bottom-right-radius:25;
 padding: 8px; 
}

#lineEdit:hover
{
border: 1px solid #f7115b;
border-left:none;
border-top-right-radius:25;
border-bottom-right-radius:25;
 padding: 8px; 
}</string>
    </property>
    <property name="placeholderText">
     <string>请输入课程名称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2
{
border:none;
color:#ffffff;
  background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #767aff,
        stop: 1 #83bcff 
    );
border-radius:25;
}


#pushButton_2:hover
{
  background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #ec7f9e,
        stop: 1 #9ca2c4 
    );
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3
{
border:1px solid  #e5e7e9;
border-right:none;
border-top-left-radius:25;
border-bottom-left-radius:25;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icons/class.png</normaloff>:/icons/class.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_4{
border:none;
border:1px solid #666765;
}

#pushButton_4::hover
{
background-color:#a7b0ca;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icons/upLoad.png</normaloff>:/icons/upLoad.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>点击上传课程图标</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>文件名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_6{
color:red;
}</string>
    </property>
    <property name="text">
     <string>上传的单张PNG图片大小不得超过400KB</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
